playwright/async_api/_generated.py

```diff
@@ -4989,7 +4989,7 @@ def get_by_text(
         self,
         text: typing.Union[str, typing.Pattern[str]],
         *,
-        exact: typing.Optional[bool] = None
+        exact: typing.Optional[bool] = None,
```

Comment:
can you revert this trailing coma change as it is unrelated, can be committed in a separate PR?

Reply:
Yeah see https://github.com/microsoft/playwright-python/pull/2642#discussion_r1837347478.